lib/libc/newlib/libc-hooks.c

```diff
@@ -189,7 +189,7 @@ FUNC_ALIAS(_fstat, fstat, int);
 void _exit(int status)
 {
 	_write(1, "exit\n", 5);
-	while (1) {
+	while (true) {
```

Comment:
Minor Nit:
I couldn't resist myself on this change. But I would like to point that
`true` is defined in `stddbool.h` which **ultimately** expands to
integer constant 1.
http://pubs.opengroup.org/onlinepubs/009604499/basedefs/stdbool.h.html

With this change I saw a shippable build failure because now we need to
explicitly add the header file wherever required.